R/dplyr_spark_connection.R

```diff
@@ -17,22 +17,43 @@ fix_na_real_values <- function(dots) {
 
 #' @export
 #' @importFrom dplyr transmute
+#' @importFrom dplyr all_of
 transmute.tbl_spark <- function(.data, ...) {
-  dots <- rlang::enquos(..., .named = TRUE) %>%
-    fix_na_real_values() %>%
-    partial_eval_dots(sim_data = simulate_vars(.data))
+  if (dbplyr_uses_ops()) {
+    dots <- rlang::enquos(..., .named = TRUE) %>%
+      fix_na_real_values() %>%
+      partial_eval_dots(sim_data = simulate_vars_spark(.data))
 
-  nest_vars(.data, dots, character())
+    nest_vars(.data, dots, character())
+  } else {
+    dots_org <- rlang::enquos(..., .named = TRUE)
```

Comment:
Unfortunately, this hack is required to use the `dbplyr` method.

Reply:
Not sure if you need me to do anything here, I'm ok with this change

Reply:
Good, just wanted to let you know.